indico/modules/events/abstracts/templates/management/abstract_list.html

```diff
@@ -73,17 +73,48 @@
                     </ul>
                 </div>
                 <div class="group">
-                    <a class="i-button icon-checkmark js-requires-selected-row js-dialog-action accept">
-                        {% trans %}Accept{% endtrans %}
+                    <a class="i-button icon-checkmark accept js-requires-selected-row"
```

Comment:
Are we really adding a button for each possible action in the end? Why not simply using the same dropdown we have on the judgment box in abstract display page?